src/handlers/testing/align/util/run-command.ts

```diff
@@ -0,0 +1,25 @@
+import { exec as execCommand } from '@actions/exec';
+
+export async function runCommandInAlignmentMode(args: {
+  command: string;
+  commandArgs: string[];
+  cliServerAddress: string;
+  testExternalId: string;
+  testCaseHash: string;
+}) {
+  const env: Record<string, string> = {
+    ...process.env,
+    AUTOBLOCKS_CLI_SERVER_ADDRESS: args.cliServerAddress,
+    AUTOBLOCKS_ALIGN_TEST_EXTERNAL_ID: args.testExternalId,
+    AUTOBLOCKS_ALIGN_TEST_CASE_HASH: args.testCaseHash,
+  };
+
+  try {
+    await execCommand(args.command, args.commandArgs, {
+      silent: true,
+      env,
+    });
+  } catch (err) {
+    // TODO handle errors
```

Comment:
What are you thinking for error handling? Log to our system and just print to standard out that something went wrong?

Reply:
we'll need to emit a CONSOLE_LOG event like we do for the `exec` command and show them in the ink app